### DISCUSSION TITLE
Plugin and Theme Marketplace

### DISCUSSION THREAD

**Discussion by silicahd** [givanz/Vvveb#44] [Ideas]
I think I always say this to people but i realized that you actually made some opencat stuff so here it is. You should make this CMS the same as opencart in terms of functionality. What I mean is keep it opensoruce but make a marketplace where people can make modules and sale them and you take a percentage of each sale.

**Comment by givanz** [comment]
Thank you for the suggestion.

Currently the marketplace is very basic it will be expanded to allow theme and plugin authors to upload their own extensions.

At a later stage a premium plugin/theme section will be availabe for authors to sell their extensions and everything will be integrated with the admin to avoid manual installation etc but there are still things to be done until this feature will be added.
